lib/components/viewers/nearby/stop-card-header.tsx

```diff
@@ -78,14 +78,16 @@ const StopCardHeader = ({
               <InvisibleA11yLabel>{zoomButtonText}</InvisibleA11yLabel>
             </button>
           ) : null}
-          <Link
-            className="pull-right"
-            style={{ color: 'inherit', fontSize: 'small' }}
-            to={actionPath}
-            toParams={actionParams}
-          >
-            <IconWithText Icon={actionIcon}>{actionText}</IconWithText>
-          </Link>
+          {actionPath && actionText ? (
+            <Link
+              className="pull-right"
+              style={{ color: 'inherit', fontSize: 'small' }}
```

Comment:
Any way we can get some hover underlines on this?

Reply:
<img width="136" alt="image" src="https://github.com/user-attachments/assets/60ac7fbd-5538-4e05-94fb-c9a97f46b074" />
??

Reply:
I moved it to a styled component anyway

Reply:
Probably a safari issue